### PR TITLE
[BEAM-1178] Make naming of logger objects consistent

### PR DESCRIPTION
**Pull request by iemejia**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [X] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [X] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [X] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by iemejia** [comment]
R: @kennknowles @dhalperi 
I added an extra commit that fixes a silly grammar error that I found too, sorry to mix it with this, but I wanted to get it fixed too (and it was too small to require its own JIRA).

File: sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java [hunk 1]
```diff
@@ -75,13 +75,13 @@
  */
 @SuppressWarnings("rawtypes")
 public class ApiSurface {
-  private static Logger logger = LoggerFactory.getLogger(ApiSurface.class);
+  private static Logger LOG = LoggerFactory.getLogger(ApiSurface.class);
```
Comment (kennknowles) [inline sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java]: Jenkins failure is just checkstyle that this one needs to be final.

**Comment by jbonofre** [comment]
CC: @jbonofre

I would use `LOGGER` as default. And the PR doesn't pass due to checkstyle (final keyword).

**Comment by iemejia** [comment]
@kennknowles Thanks for noticing, I don't know how I miss this one, sorry, it should be ok now.

@jbonofre I chose LOG because I wanted to fix the issue but at the same time to touch the less amount of classes possible, so after running some stats in master I found that:

5   files use LOGGER
9   files use logger
100 files use LOG

So for me the natural way to go is to choose 'LOG'.

**Comment by jbonofre** [comment]
Fair enough but I think LOGGER us more natural as the slf4j class is LOGGER.

**Comment by kennknowles** [comment]
LGTM. I am happy with either name. Technically I prefer all caps to be reserved for things that are "deeply immutable" aka really just values. But `LOG` and `LOGGER` are both very common anyhow. I'll merge this and renames in the future are fine, too.

**Pull request merged**